testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -1933,4 +1932,2003 @@ public void verifyFT21() throws IOException, InterruptedException {
+                EntityUtils.toString(response.getEntity()));
+    }
+
+    /**
+     * test DeactivateSubscription API with Blank Params
+     *
+     * JIRA issue: https://applab.atlassian.net/browse/NMS-193
+     */
+    @Ignore
+    @Test
+    public void verifyFT105() throws IOException, InterruptedException {
+
+        Subscriber subscriber = subscriberService.getSubscriber(1000000000L);
+        Subscription subscription = subscriber.getActiveSubscriptions()
+                .iterator().next();
+        String subscriptionId = subscription.getSubscriptionId();
+
+        // callingNumber blank
+        HttpDeleteWithBody httpDelete = createDeactivateSubscriptionHttpDelete(
+                "", "A", "AP", "123456789012345", subscriptionId);
+        String expectedJsonResponse = createFailureResponseJson("<callingNumber: Not Present>");
+        HttpResponse response = SimpleHttpClient.httpRequestAndResponse(httpDelete, ADMIN_USERNAME, ADMIN_PASSWORD);
+
+        assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusLine().getStatusCode());
+        assertEquals(expectedJsonResponse, EntityUtils.toString(response.getEntity()));
+
+        // callingNumber blank
+        httpDelete = createDeactivateSubscriptionHttpDelete(" ", "A", "AP",
+                "123456789012345", subscriptionId);
+        response = SimpleHttpClient.httpRequestAndResponse(httpDelete,
+                ADMIN_USERNAME, ADMIN_PASSWORD);
+
+        assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusLine()
+                .getStatusCode());
+        assertEquals(expectedJsonResponse,
+                EntityUtils.toString(response.getEntity()));
+    }
+
+    /**
+     * To verify that Deactivate Subscription Request API request fails if the
+     * provided parameter value of circle is : empty value.
+     */
+    @Test
+    public void verifyFT106() throws IOException, InterruptedException {
+
+        Subscriber subscriber = subscriberService.getSubscriber(1000000000L);
+        Subscription subscription = subscriber.getActiveSubscriptions()
+                .iterator().next();
+        String subscriptionId = subscription.getSubscriptionId();
+        // circle blank
+        HttpDeleteWithBody httpDelete = createDeactivateSubscriptionHttpDelete(
+                "1000000000", "A", "", "123456789012345", subscriptionId);
+
+        HttpResponse response = SimpleHttpClient.httpRequestAndResponse(
+                httpDelete, ADMIN_USERNAME, ADMIN_PASSWORD);
+
+        assertEquals(HttpStatus.SC_OK, response.getStatusLine()
+                .getStatusCode());
+
+        // circle blank(single space)
+        httpDelete = createDeactivateSubscriptionHttpDelete("1000000000", "A",
+                " ", "123456789012345", subscriptionId);
+
+        response = SimpleHttpClient.httpRequestAndResponse(httpDelete,
+                ADMIN_USERNAME, ADMIN_PASSWORD);
+
+        assertEquals(HttpStatus.SC_OK, response.getStatusLine()
```

Comment:
As per New Api doc version 1.6 , this is successful scenario. 
As soon as FT doc is changed, I will update the description.